caikit_nlp/modules/text_embedding/embedding.py

```diff
@@ -352,17 +366,20 @@ def run_sentence_similarity(
             SentenceSimilarityResult: Similarity scores for each sentence.
         """
 
-        source_embedding = self._encode_with_retry(
+        source_embedding, source_token_count = self._encode_with_retry(
             source_sentence, truncate_input_tokens=truncate_input_tokens
         )
-        embeddings = self._encode_with_retry(
+        embeddings, embeddings_token_count = self._encode_with_retry(
```

Comment:
nit: embeddings_token_count is a confusing var name. 

sentences_token_count or well even count2 would be better "embeddings_token_count", but doesn't matter. Varable name can be fixed whenever.

Reply:
Renaming to `sentences_token_count`.